packages/create-redwood-app/README.md

```diff
@@ -27,16 +31,23 @@ Explains how to contribute by addressing the following three points:
 
 ### How to run create redwood-app script locally
 
-After making changes to `create-redwood-app.js`:
+Install [`esm`](https://www.npmjs.com/package/esm);
 
+```bash
+yarn workspace add --dev create-redwood-app
 ```
-node -r esm ./redwood/packages/create-redwood-app/src/create-redwood-app.js ../path/to/new-project`
-```
 
-If running from within the current redwood source directory, be sure your target app path is outside.
+After making changes to `create-redwood-app.js`, do one of the following;
+
+```bash
+# Run and install yarn dependencies
+node -r esm ./packages/create-redwood-app/src/create-redwood-app.js ../create-redwood-app-test
 
-Note: Assumes that
+# Run without installing yarn dependencies
+node -r esm ./packages/create-redwood-app/src/create-redwood-app.js --no-yarn-install ../create-redwood-app-test
 
+# For the lazy; forcefully delete ../create-redwood-app-test and then run without installing yarn dependencies
+rm -rf ../create-redwood-app-test && node -r esm ./packages/create-redwood-app/src/create-redwood-app.js --no-yarn-install ../create-redwood-app-test
 ```
-npm install --save esm
-```
\ No newline at end of file
+
+_Note: In above example we are creating the app in the parent directory in folder `create-redwood-app-test`._
```

Comment:
These instructions could potentially trip you up; I would suggest doing the following instead:

1. Run `yarn build:watch` command which will transpile your code when it's modified.
2. Run `node ./dist/create-redwood-app.js` to test.

Reply:
@peterp Much better! I'll update the docs.